client/modules/IDE/pages/IDEView.jsx

```diff
@@ -370,41 +370,43 @@ class IDEView extends React.Component {
                 <header className="preview-frame__header">
                   <h2 className="preview-frame__title">Preview</h2>
                 </header>
-                <div className="preview-frame-overlay" ref={(element) => { this.overlay = element; }}>
+                <div className="preview-frame__content">
+                  <div className="preview-frame-overlay" ref={(element) => { this.overlay = element; }}>
```

Comment:
this is probably a task for a separate issue/PR but i was looking at this and maybe this overlay thing isn't necessary anymore. i added it to make the pane resizing work. the dragging resizer thing used to be positioned over the iframe, and the overlay was necessary for the iframe not to capture the mouse events, but then #441 happened, and it got moved to be between the code editor and the preview.